integration_tests/publish-test-playground/gradle.properties

```diff
@@ -0,0 +1,19 @@
+android.useAndroidX=true
+
+# latest
+agpVersion=7.1.0
```

Comment:
@ZSmallX as we discussed offline, could you try updating project level dependency versions instead of creating a new playground module?

Reply:
Sure, here are 2 things we can do:  
1. Updating project level dependency versions, like kotlin 1.4.31 -> 1.6.10, to make it suitable for both integration test and building.
2. Add a separated user-end integration tests based on artifacts deployment besides developer-end integration tests based on build system. 

Those two testing levels could have different configurations and impacts on verification and building and testing.

Is there another playground for user-end integration tests or we add a separated playground in this project for reusable convenience or we don't need such a verification convenience and just make sure it works under the latest versions?